package.json

```diff
@@ -3,20 +3,188 @@
   "version": "1.0.0",
   "description": "This extension provides a react Admin UI for managing custom fields related to [DCAT-US Schema](https://https://resources.data.gov/resources/dcat-us/).",
   "main": "index.js",
-  "devDependencies": {
-    "chance": "^1.1.6",
-    "cypress": "^4.11.0",
-    "cypress-file-upload": "^4.1.1"
-  },
   "scripts": {
+    "start:metadata-app": "node react-scripts/start.js",
+    "build:metadata-app": "node react-scripts/build.js",
+    "test:metadata-app": "node react-scripts/test.js",
+    "pre-commit": "echo 'formatting your changes...' && eslint metadata-app/src/ && prettier --write",
     "e2e": "yarn dockerize && yarn && npx cypress run",
     "dockerize": "docker-compose -f docker-compose.yml -f docker-compose.seed.yml up -d",
-    "build": "yarn build:metadata-app && yarn move:builds && yarn rename:builds",
-    "build:metadata-app": "cd metadata-app && yarn && yarn build && cd ..",
-    "move:builds": "rm -rf ckanext/dcat_usmetadata/public && mkdir -p ckanext/dcat_usmetadata/public && cp -r metadata-app/build/static/* ckanext/dcat_usmetadata/public/",
```

Comment:
@kmanaseryan how it works now? I can't see any replacement for it.

Reply:
Now since we have direct control to the build files, it builds directly to the `ckanext/dcat_usmetadata/public` folder with relevant file names.